changelog.d/20250218_192952_mzhiltso_project_quality.md

```diff
@@ -0,0 +1,4 @@
+### Added
+
+- Quality analytics for projects
```

Comment:
For the target user this patch does not introduce quality analytics for projects.
It only adds quality settings on a project level